python/lsst/daf/butler/registry/queries/exprParser/parserYacc.py

```diff
@@ -256,6 +344,15 @@ def p_literal_str(self, p):
         """
         p[0] = StringLiteral(p[1])
 
+    def p_literal_time(self, p):
+        """ literal : TIME_LITERAL
```

Comment:
This docstring looks odd but I see it matches the others.

Reply:
`ply` uses docstrings to define grammar rules, so it does look different from LSST standard.